### PR TITLE
fix(refs  dplan-12767): Make create statement work again

### PR DESCRIPTION
**Pull request by salisdemos**
### Ticket
DPLAN-12767

This is part of the Vue3 Migration https://github.com/demos-europe/demosplan-core/pull/1151

The validation for a dropdown (dpmultiselect used as single select) didn't work. So I decided to swap it with a  simple dpSelect. For that the datastructure had to be changed a little

### CODE REVIEW

File: client/js/components/assessmenttable/DpNewStatement.vue [hunk 1]
```diff
@@ -230,8 +248,9 @@ export default {
     },
 
     setPhaseValue (value) {
+      console.log('setPhaseValue', value)
```
Comment (elmasdemos) [inline client/js/components/assessmenttable/DpNewStatement.vue:251]: please remove this

Comment (salisdemos) [inline client/js/components/assessmenttable/DpNewStatement.vue:251]:
```suggestion
```

Comment (salisdemos) [reply client/js/components/assessmenttable/DpNewStatement.vue]: thanks. https://github.com/demos-europe/demosplan-core/pull/3916/commits/9a6d41541440e53072e864d94291c135b606a9d7

**Review by elmasdemos — APPROVED** [review]
thanks for the fast change. it looks fine now. lgtm, code review

**Pull request merged**